go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.8.4
+	github.com/tonistiigi/dchapes-mode v0.0.0-20241001053921-ca0759fec205
```

Comment:
I see the original repository has a number of tagged versions already; should those be inherited? https://hg.sr.ht/~dchapes/mode/tags

Reply:
This is not the same source. I don't plan to make my own releases in the fork.